pyanaconda/modules/common/structures/storage.py

```diff
@@ -453,3 +453,61 @@ def get_root_device(self):
         :return: a device name or None
         """
         return self.mount_points.get("/")
+
+
+class PlatformMountPointData(DBusData):
```

Comment:
The same structure can eventually used to expose the autopart specification. Please, take it into account when choosing a name of this structure.

Reply:
I am not sure what would be the best name for this, I changed it to `RequiredMountPointData` and added a better docstring (or at least I hope it is better).